### PR TITLE
Add keyboard 7skb

### PR DESCRIPTION
**Pull request by Salicylic-acid3**
Add 7skb keyboard.

A 63 keys HHKB Layout keyboard.
Salicylic-acid3

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/7skb/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/7skb/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  TG(_ADJUST), XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   RESET, \
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------+--------+--------+--------|
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     RGB_RST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------+--------+--------|
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     RGB_TOG, RGB_MOD, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------+--------|
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     RGB_VAD, RGB_VAI, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, XXXXXXX, \
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------+--------|
+               XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,              XXXXXXX, XXXXXXX,          KC_STOP, XXXXXXX  \
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/7skb/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          //`---------------------------------------------|   |--------------------------------------------'
+  )
+};
+
+
+//A description for expressing the layer position in LED mode.
+uint32_t layer_state_set_user(uint32_t state) {
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/default/keymap.c]:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

File: keyboards/7skb/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+#ifdef RGBLIGHT_ENABLE
+    switch (biton32(state)) {
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/default/keymap.c]:
```suggestion
    switch (get_highest_layer(state)) {
```

File: keyboards/7skb/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+      result = true;
+      break;
+  }
+
+  return result;
+}
+
+void matrix_init_user(void) {
+
+}
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/default/keymap.c]: This can be deleted.

File: keyboards/7skb/keymaps/salicylic/keymap.c [hunk 6]
```diff
@@ -0,0 +1,186 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/7skb/keymaps/salicylic/keymap.c]:
```suggestion
```

File: keyboards/7skb/readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# 7skb
+
+![7skb](https://cdn-ak.f.st-hatena.com/images/fotolife/S/Salicylic_acid3/20191124/20191124025208.png)
+
+This is 63 keys Custom keyboard.
+
+Keyboard Maintainer: [Salicylic_acid3](https://github.com/Salicylic-acid3)
```
Comment (fauxpark) [inline keyboards/7skb/readme.md]:
```suggestion
* Keyboard Maintainer: [Salicylic_acid3](https://github.com/Salicylic-acid3)
```

File: keyboards/7skb/readme.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+Hardware Supported: 7skb PCB, Pro Micro
```
Comment (fauxpark) [inline keyboards/7skb/readme.md]:
```suggestion
* Hardware Supported: 7skb PCB, Pro Micro
```

File: keyboards/7skb/readme.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+Hardware Availability: [PCB & Case Data](https://github.com/Salicylic-acid3/PCB_Data), [Booth Shop](https://salicylic-acid3.booth.pm/items/1673395)
```
Comment (fauxpark) [inline keyboards/7skb/readme.md]:
```suggestion
* Hardware Availability: [PCB & Case Data](https://github.com/Salicylic-acid3/PCB_Data), [Booth Shop](https://salicylic-acid3.booth.pm/items/1673395)
```

File: keyboards/7skb/rev1/rev1.c [hunk 10]
```diff
@@ -0,0 +1,5 @@
+#include "7skb.h"
+
+void matrix_init_kb(void) {
+	matrix_init_user();
+};
```
Comment (fauxpark) [inline keyboards/7skb/rev1/rev1.c]: This can be deleted.

File: keyboards/7skb/rev1/rev1.h [hunk 11]
```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "7skb.h"
+
+#include "quantum.h"
+#ifdef RGBLIGHT_ENABLE
+//rgb led driver
+#include "ws2812.h"
+#endif
```
Comment (fauxpark) [inline keyboards/7skb/rev1/rev1.h]: Not sure this include is needed.

Comment (Salicylic-acid3) [reply keyboards/7skb/rev1/rev1.h]: Thank you for pointing out.
I fixed it.

File: keyboards/7skb/rev1/rev1.c [hunk 12]
```diff
@@ -0,0 +1,2 @@
+#include "7skb.h"
```
Comment (noroadsleft) [inline keyboards/7skb/rev1/rev1.c]: This should actually be

```suggestion
#include "rev1.h"
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**